website/docs/guides/message-extraction.md

```diff
@@ -5,41 +5,54 @@ description: Learn about message extraction in i18n and how to use Lingui to ext
 
 # Message Extraction
 
-Message extraction is an essential step in the internationalization process. It involves analyzing your code and extracting all messages defined in it so that your message catalogs are always up-to-date with the source code.
+Message extraction is a key part of the internationalization process. It involves scanning your codebase to identify and extract all the defined messages, ensuring that your message catalogs stay synchronized with the source code.
 
-To extract messages (as marked with `<Trans>`, `t` or other macros) from your application, use the `lingui extract` cli command.
+In practice, developers define messages directly in the source code, and the extraction tool automatically collects these messages and stores them in a message catalog for translation.
 
-## Supported patterns
+Read more about the [`lingui extract`](/docs/ref/cli.md) command.
 
-The extractor operates on a static level and doesn't execute your code. As a result, complex patterns and dynamic code are not supported.
+## Supported Patterns
 
-### Macro usages
+The extractor operates at a static level, meaning that it analyzes the source code without executing it. As a result, it doesn't support complex patterns or dynamic code, and only simple, statically defined messages are collected.
 
-Extractor supports all macro usages, such as the following examples:
+### Macros Usage
+
+> Macros are JavaScript functions that run at build time. The value returned by a macro is inlined into the bundle instead of the original function call.
```

Comment:
```suggestion
> Macros are JavaScript transformers that run at build time. The value returned by a macro is inlined into the bundle instead of the original function call.
```

maybe this is a better word?